File: Program37.c
//C Program to Perform Addition of Non Factors
#include<stdio.h>

int SumNonFactors(int iNo)
{
    int iCnt = 0 ;
    int iSum = 0 ;

    for(iCnt = 1 ; iCnt < iNo ; iCnt++)
    {
        if((iNo % iCnt)!= 0 )
        {
            iSum = iSum + iCnt ;
        }
    }
    return iSum ;


}

int main()
{
    int iValue = 0 ;
    int iAns = 0 ;
    
    printf("Enter the Number\n");
    scanf("%d",&iValue);

    iAns = SumNonFactors(iValue);

    printf("Sum of Non Factors are :%d " , iAns);

    return 0 ;
}

File: Program37.py
# Input : 4 
# Output : 
# 4 
# 3 
# 2 
# 1 

def displayPattern(iValue) : 
    for i in range(  1 , iValue + 1 , 1 ):
        print(i,"\t*\t",end = " " ) 

def main():
    iNo = 0  

    iNo = int(input("Enter a Number : ")) 

    displayPattern(iNo)

if __name__=="__main__":
    main()